Pspcl/Pspcl.Services/StockService.cs

```diff
@@ -506,8 +522,83 @@ public void UpdateStockMaterial(List<List<int>> selectedRowsToDelete)
             }
             _dbcontext.SaveChanges();
             
-        }       
+        }
+
+        public string UploadImageToAzure(IFormFile file)
```

Comment:
Initially, I told it to be a separate service to handle all blob related functions and should be generic as well. So that later we can use this upload functionality simply on other pages.